diff -up GraphicsMagick-1.3.31/PerlMagick/Makefile.PL.in.0000~ GraphicsMagick-1.3.31/PerlMagick/Makefile.PL.in
--- GraphicsMagick-1.3.31/PerlMagick/Makefile.PL.in.0000~	2018-11-27 18:39:24.640152647 +0100
+++ GraphicsMagick-1.3.31/PerlMagick/Makefile.PL.in	2018-11-27 18:40:16.279444315 +0100
@@ -78,7 +78,7 @@ WriteMakefile
    'INSTALLBIN' => $magick_BIN_DIR,
 
    # Library specification
-   'LIBS'       => ["-L$magick_LIB_DIR $magick_LDFLAGS $magick_API_LIBS"],
+   'LIBS'       => ["-L$magick_LIB_DIR -L../magick/.libs $magick_LDFLAGS $magick_API_LIBS"],
 
    # Perl binary name (if a Perl binary is built)
    'MAP_TARGET' => 'PerlMagick',
